aiida/backends/tests/cmdline/commands/test_export.py

```diff
@@ -155,18 +141,17 @@ def test_create_tar_gz(self):
         finally:
             delete_temporary_file(filename)
 
-    @unittest.skip("Reenable when issue #2426 has been solved (migrate exported files from 0.3 to 0.4)")
     def test_migrate_versions_old(self):
         """Migrating archives with a version older than the current should work."""
         archives = [
-            'export_v0.1.aiida',
-            'export_v0.2.aiida',
-            'export_v0.3.aiida'
+            'export_v0.1_no_UPF.aiida',
```

Comment:
just pointing out that the meaning of `no_UPF` might become unclear to people once atomic stuff has been migrated out of aiida-core.
up to you whether you want to keep this label.

Reply:
Yeah, I don't want to keep this label.

They will be renamed to `simple` instead.

Reply:
Why is this marked as resolved?

Reply:
Ok, I guess I'm not yet supposed to look at the PR - just ping me when I should check it again.

Reply:
Reloved it because I changed it to `_simple`. But let's keep it unresolved if there should be an uproar for the label change.

Reply:
should I check it again?

Reply:
This particular thing, or the updated code in general?

If it's the first, no.
If it's the second, I guess, if you want to, yes? :)